lib/builtin.pm

```diff
@@ -134,7 +134,7 @@ bundles.
 
     $val = false;
 
-Returns the boolean fiction value. While any non-true scalar value is
+Returns the boolean false value. While any non-true scalar value is
```

Comment:
I think the documentation for `true` above should be changed to match too, i.e. "the boolean true value."